__tests__/app.test.js

```diff
@@ -208,24 +211,34 @@ describe("POST /api/articles/:article_id/comments", () => {
       body: "This is a gandalf comment for article_id 4",
     };
     return request(app)
-      .post("/api/articles/1/comments")
+      .post("/api/articles/4/comments")
       .send(newComment)
-      .expect(400)
+      .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Bad request, please provide valid input")
+        expect(body.msg).toBe("Request parameter not found");
```

Comment:
I think this should actually be bad request - since our username has to be a string in our database.